packages/contracts-bedrock/tasks/update-registry.ts

```diff
@@ -0,0 +1,62 @@
+import fs from 'fs'
```

Comment:
I wouldn't be opposed to using a foundry script for this, we started adding them to the `scripts` directory

Reply:
I think probably easiest to keep as hardhat and move all the scripts over to foundry later